File: prime.py
# Program Starts

primeNumber = input("Enter your number: ")

if primeNumber is 1 or primeNumber is 0:
    print "Neither composite nor prime."
elif primeNumber is 2 or primeNumber is 3:
    print "%(primeNumber)s is a prime number." % locals()
elif primeNumber % 2 is 0 or primeNumber % 3 is 0:
    print "%(primeNumber)s is a composte number." % locals()
else:
    print "%(primeNumber)s is a prime number." % locals()

File: prime.php
<?php
  // Program Starts

  echo "Enter your number: ";

  $primeNumber = fgets(STDIN);

  switch (true) {
    case $primeNumber == 1 || $primeNumber == 0:
      echo "Neither composite or prime.";
      break;
    case $primeNumber == 2 || $primeNumber == 3;
      echo "$primeNumber is a prime number.";
      break;
    case $primeNumber % 2 == 0 || $primeNumber % 3 == 0:
      echo "$primeNumber is a composite number.";
      break;
    default:
      echo "$primeNumber is a prime number.";
  };
?>
